pkg/kubelet/pod_workers.go

```diff
@@ -1059,6 +1176,25 @@ func (p *podWorkers) removeTerminatedWorker(uid types.UID) {
 	delete(p.podSyncStatuses, uid)
 	delete(p.podUpdates, uid)
 	delete(p.lastUndeliveredWorkUpdate, uid)
+
+	fullname := status.fullname
+	if p.startedStaticPodsByFullname[fullname] == uid {
+		delete(p.startedStaticPodsByFullname, fullname)
+	}
+	l := 0
```

Comment:
can be removed as per https://github.com/kubernetes/kubernetes/pull/104743/commits/e77d81267795d914ef7ebab7e13c6a6baf986971#r720358696